### PR TITLE
skypeforlinux-8.90.0.407-r1.ebuild

### PR DESCRIPTION
**Pull request by rahul-singh-sandhu**
Version bump for skype for linux due to 404.

Signed-off-by: Rahul Sandhu <rahul@sandhuservices.dev>

### CODE REVIEW

**Comment by rahul-singh-sandhu** [comment]
Bug 882285: https://bugs.gentoo.org/882285

**Comment by scardracs** [comment]
You have forgot to run `pkgdev manifest`

File: net-im/skypeforlinux/skypeforlinux-8.90.0.407-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	dev-libs/nspr[${MULTILIB_USEDEP}]
+	dev-libs/nss[${MULTILIB_USEDEP}]
+	media-libs/alsa-lib[${MULTILIB_USEDEP}]
+	media-libs/fontconfig:1.0[${MULTILIB_USEDEP}]
+	media-libs/freetype:2[${MULTILIB_USEDEP}]
+	media-libs/libv4l[${MULTILIB_USEDEP}]
+	net-print/cups[${MULTILIB_USEDEP}]
+	sys-apps/dbus[${MULTILIB_USEDEP}]
+	sys-devel/gcc[cxx]
+	sys-libs/glibc
```
Comment (scardracs) [inline net-im/skypeforlinux/skypeforlinux-8.90.0.407-r1.ebuild:39]: I'm not sure calling gcc and glibc is required anymore but better to ask to someone more qualified

**Comment by rahul-singh-sandhu** [comment]
@scardracs Manifest done :) It failed to build without gcc and glibc for me.

File: net-im/skypeforlinux/skypeforlinux-8.90.0.407-r1.ebuild (hunk 1 above)
Comment (thesamesam) [reply net-im/skypeforlinux/skypeforlinux-8.90.0.407-r1.ebuild:39]: Just leave it in. gcc is needed for libstdc++ here and glibc is needed as it won't work on musl.

**Comment by scardracs** [comment]
Did you run pkgdev manifest as I said? I see too many lines inside Manifest. It should have to be 

```
DIST skypeforlinux_8.85.0.409-1.x86_64.rpm 125943432 BLAKE2B 7300c53c85387df71a7b1ccdd56a12a659a0e8ddbf8b274c32ac956e0bd67269815bc56e4d3a91cbc3f8214a3259026147ae3e8467efec934d06424495bf502d SHA512 2ee36ea93259390bb1061aae5ea86ac5617ac411f058b62607324fc36debbd7c914e872b9a2741ff72b37bc5e1b4683d862d3c01b84f9a3fe98de2e20959c911
DIST skypeforlinux_8.86.0.407-1.x86_64.rpm 126036696 BLAKE2B e9fb9fb5c3612c1818d801c1ce43848ca2e313a7d09009678b6aeb1a649cd29c58a6e798fc22e48a6d23ceedd3a99b087110abc7b3d4cb8ba97cbe31d1f77c2e SHA512 8546a0bfc367e801b0f94d926dedf87dd2212888ea4d9206b1ed650c334ceebc1d72029a81c826dbf1630ec6625a6e08c7ae942e6dd79403027671996385435b
DIST skypeforlinux_8.87.0.403-1.x86_64.rpm 126049259 BLAKE2B e094fcda81bff2509a0411d12758fd3dfcaaf57ea56ac02c0243a13ea8c1a84c3558ec8c67a33bef4995d8c18f4d3df52d85f5971cb60d6e080f43d4fa124a35 SHA512 80d3ddd384bea9fb1cea3d802da4e65acd441656d794728071879476160b6fbcf22d17152e8d06b0d3250596e8ff77de249d502db89484b20b15e1ef56f1e01c
DIST skypeforlinux_8.90.0.407-1.x86_64.rpm 126406385 BLAKE2B ef3627e786b60a512576a6cee5af8b53c10a54122cb2de784a534c5effb13ea053070796a52c6672344dc5abb6dc3b503e89a1f14960c8990173b21acfc0b8e2 SHA512 ad74e6e8006f23c2f70f67373fa386e19e3b9e4da5e1ab29956191bb3626099d06ebcc80ace458401dd4f4a62d894eb735f7be0a60b6f668d576aaadf119d39d

```

**Comment by listout** [comment]
@rahul-singh-sandhu Other devs have already suggested what needs to be done. Might as well fix the commit message, we usually do ```foo/bar: Version bump to x.y.z``` and maybe squash commits.

File: net-im/skypeforlinux/Manifest [hunk 2]
```diff
@@ -2,3 +2,10 @@ DIST skypeforlinux_8.83.0.411-1.x86_64.rpm 140568859 BLAKE2B 3e429e5c60fe812092a
...
+EBUILD skypeforlinux-8.87.0.403-r1.ebuild 3250 BLAKE2B eed5437754666a9da47304ab1da0539bb8e3e0e7f7e462b7e0de527038cc024f7f0d3c513ffcaaacac9d4c3223de6aa07b26f14f00d2bfa5b9ee872857909eb2 SHA512 09758ed08729dbb173e6d03fc0dbc1e04f3271ba42ff1e837d73bddb88a07a55fe67d64bfd59986a3af67e39d11e170459a2e92e1cf51f7f911cbc3f6b3bf55e
+EBUILD skypeforlinux-8.90.0.407-r1.ebuild 3250 BLAKE2B eed5437754666a9da47304ab1da0539bb8e3e0e7f7e462b7e0de527038cc024f7f0d3c513ffcaaacac9d4c3223de6aa07b26f14f00d2bfa5b9ee872857909eb2 SHA512 09758ed08729dbb173e6d03fc0dbc1e04f3271ba42ff1e837d73bddb88a07a55fe67d64bfd59986a3af67e39d11e170459a2e92e1cf51f7f911cbc3f6b3bf55e
+MISC metadata.xml 461 BLAKE2B 9bcbfe1d6892c40ce917a76c600baad338b5b86967e5656f79b9bf29e75c3ba474dc11444d8e347769f37934610a971097f96b22ae171ea46cd3f1aef2e9f3e5 SHA512 81df3ef175eafabad357f3d5792c71534b85ffd2c1bd9280ea7ef9cbea17aceb396bc7ac65abdb09e5fba8c3d98384c0d67c58a8daa2d42a657e1a7b150b52e7
```
Comment (scardracs) [inline net-im/skypeforlinux/Manifest:11]: Please use pkgdev manifest to update it, because these should not be here

**Comment by listout** [comment]
@rahul-singh-sandhu @scardracs I think we can close this with [5cbd4b9dd37a7dac4e939e4543ea0351b9f1d7da](https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=5cbd4b9dd37a7dac4e939e4543ea0351b9f1d7da) being merged.

**Comment by scardracs** [comment]
Yep you can

**Pull request closed without merging**